HS_Save_Editor/flags.txt

```diff
@@ -254,24 +253,24 @@
 6.33.18:gem-gem-gem
 6.30.12:gdoor-gdoor-gdoor
 6.24.13:gkey-gkey-gkey
-23.97.18:DoesNotExist-lava_charm-lava_charm
```

Comment:
I know the things in the caves technically exists in NG but the map is not accessible so I think these should be marked does not exist